wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
 # Privacy Statement
 
-Please note that the English version of this privacy statement is the original and, as such, the binding version. To access the English version, go to the top/top right of this page and click on the "en" link.
-The following only applies to the version you can download from this website, [jamulus.io](https://jamulus.io). 
+**The English version of this document represents the official privacy statement. Should a conflict arise between the English language version and any translation hereof, the English version shall prevail. This statement applies to the _current version_ of the Jamulus software. Earlier versions may have had different privacy terms which are no longer applicable. Users are encouraged to use the most current version of the Jamulus software.**
+ 
 
 ## Definition of Terms
 
-- "**Server**" The Jamulus Server software, as opposed to the host machine/OS it is running on.
-- "**Client**" The Jamulus software used to connect to a Server
+- "**Server**" The Jamulus Server software running on a remote computer to which you will connect.
+- "**Client**" The Jamulus software running on your local computer used to connect to a Server
 - "**Directory**" A Jamulus Server configured to supply a list of Servers to Clients
 
-## Web site
+## Jamulus.io Web site
 
-The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. We do not collect your personal data or set tracking cookies. When you connect to the jamulus.io site, your IP is sent to [GitHub Pages](https://pages.github.com/).
+The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. See Github's [privacy policy](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement) for information relating to data collection and privacy.
```

Comment:
I'd like to have the sentence "We do not set tracking cookies" or something comparable here. At least it should be clear that the Jamulus Team does nothing privacy critical on the Website.